builtin/logical/pki/path_root.go

```diff
@@ -231,6 +231,7 @@ func (b *backend) pathCAGenerateRoot(ctx context.Context, req *logical.Request,
 	resp.Data["issuer_id"] = myIssuer.ID
 	resp.Data["issuer_name"] = myIssuer.Name
 	resp.Data["key_id"] = myKey.ID
+	resp.Data["key_name"] = myKey.Name
```

Comment:
Do we return `key_name` on the fetch APIs?

Reply:
We do not on the fetch issuer api.